internal/sgbouncewizard/sgbouncewizard.go

```diff
@@ -577,6 +570,50 @@ func (srv *Server) Serve(Port int) {
 		})
 	})
 
+	r.Route("/ws", func(r chi.Router) {
+		r.Get("/", func(w http.ResponseWriter, r *http.Request) {
+			m.HandleRequest(w, r)
+		})
+	})
+
+	m.HandleMessage(func(s *melody.Session, msg []byte) {
+		log.Println(string(msg))
+		params := strings.Split(string(msg), " ")
+		lock.Lock()
+		if params[0] == "edit" {
```

Comment:
Can you break out the various portions of your message into variables so its easier to tell what it is? It isn't immediately apparent what `params[0]`, `params[1]`, etc... are

Also if something happens during communication and things don't split like you intend then these will segfault if you try to reference `params[1]`, so I'd add some input validation too.

Reply:
Yeah that's my bad, there's a corresponding frontend PR that I made along with this that should've gave more clarification

Reply:
`param[0]` is intended to be the command that the client is issuing
`param[1]` is the ID of the bounce rule that the command is going to be issued on